### PR TITLE
Merge release/3.5 back to develop

### PR DESCRIPTION
**Pull request by wolf31o2**
Includes one commit/PR:

Add cdap-security-extn and apache-sentry repos, update refs (#9)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,23 +2,44 @@
...
 
-This checks out CDAP, Cask Tracker, CDAP Navigator, and Hydrator
-Plugins. The external repositories are located under
-`app-artifacts`, so CDAP should be built using
-`-Dadditional.artifacts.dir=$(pwd)/app-artifacts` to include the
-additional projects in the CDAP Master packages. Compiling the
-artifacts requires first building and installing the CDAP API JARs into
-your local Maven repository.
+This checks out CDAP, Cask Tracker, CDAP Navigator, CDAP Security
+Extensions, and Hydrator Plugins. The external application artifact
```
Comment (awholegunch) [inline README.md]: Cask Hydrator

File: README.md [hunk 2]
```diff
@@ -2,23 +2,44 @@
...
+extensions are located under `security-extensions`, so CDAP should
+be built using
+`-Dadditional.artifacts.dir=$(pwd)/app-artifacts -Dsecurity.extensions.dir=$(pwd)/security-extensions`
+to include the additional projects in the CDAP Master packages.
+
+## Compiling/Installing Apache Sentry 1.7.0
+
+The CDAP Security Extensions requires you to have Apache Sentry 1.7.0
+JARs in your local Maven repository. These JARs are not available from
+Maven Central, so you may need to compile them, yourself. The correct
```
Comment (awholegunch) [inline README.md]: so you may need to compile them yourself.

File: README.md (hunk 1 above)
Comment (wolf31o2) [inline README.md]: This is a merge. To update this, I have to create another PR for `release/3.5` and have that merged, first. Can we do it afterwards, instead?

File: README.md [hunk 3]
```diff
@@ -2,23 +2,44 @@
...
+
+```bash
+git submodule init
+git submodule update
+cd apache-sentry
+mvn clean install -BskipTests
+cd ..
+```
 
 ## Installing CDAP API JARs:
```
Comment (awholegunch) [inline README.md]: ## Installing CDAP API JARs

File: README.md [hunk 4]
```diff
@@ -2,23 +2,44 @@
...
+Compiling the artifacts requires first building and installing the
+CDAP API JARs into your local Maven repository.
+
 ```bash
 export MAVEN_OPTS="-Xmx3056m -XX:MaxPermSize=128m"
-mvn install -DskipTests -B -am -pl cdap/cdap-api,cdap/cdap-app-templates/cdap-etl/cdap-etl-api -P templates
+mvn install -DskipTests -B -am -pl cdap/cdap-api,cdap/cdap-app-templates -P templates
 ```
 
 ## Compiling CDAP + Cask Tracker + CDAP Navigator + Hydrator Plugins (example)
```
Comment (awholegunch) [inline README.md]:
```## Compiling CDAP + Cask Tracker + CDAP Navigator + Cask Hydrator Plugins (example)

File: README.md (hunk 2 above)
Comment (wolf31o2) [inline README.md]: Umm... why is that? It's fine how I wrote it. Rather than get into a comma discussion. I'm removing the unnecessary identifier.

**Comment by awholegunch** [comment]
LGTM

**Pull request merged**